app/lib/ability.rb

```diff
@@ -290,6 +257,12 @@ def facility_senior_staff_abilities(user, resource, controller)
 
 
   def facility_staff_abilities(user, resource, controller)
+    cross_core_abilities(user, resource, controller)
+
+    if resource.is_a?(OrderDetail) && user.facility_staff_or_manager_of?(resource.facility)
+      can :manage, OrderDetail, order: { facility_id: resource.order.facility_id }
```

Comment:
I'm not sure but this might be giving staff more abilities than they want/realize. Let's push this to stage and call it out in the testing notes